pkg/kubernetes/client.js

```diff
@@ -680,6 +697,34 @@ define([
         });
 
         /*
+         * add_watcher:
+         * @type: The type of watch to add
+         *
+         * Returns a Promise that if successfull will
+         * be a KubernetesWatch object for the given type.
+         * The same object will be available at self.watches[type]
+         */
+        self.add_watcher = function add_watcher (type) {
```

Comment:
Can we call this function client.include(kind) ... and make it driven off of kind instead of type? That would make it clear, IMO what is expected to happen.